### DISCUSSION TITLE
更新lantern版本后无法登录

### DISCUSSION THREAD

**Discussion by Dionysus199404** [getlantern/lantern-forum-cn#628] [Q&A]
windows11 ，mac os15  更新lantern版本后，目前都无法通过邮箱登录，手机上5开头吧版本的lantern，无法正常使用，有账号信息，但是无法给其他设备授权

账号邮箱18163658757@163.com

**Comment by ChaoQunPeng** [comment]
我也是，通过电子邮箱登录，点击提交按钮没有反应

**Comment by MaxMKMax** [comment]
@Dionysus199404 @ChaoQunPeng 
您好，谢谢您的反馈。蓝灯已经在正视这个问题了。 我们正在修复问题和进行优化。请您耐心等待。非常抱歉影响了您的体验感。谢谢您的等待与支持。

您可以帮助我们在应用程序中提交报告您的问题吗？您可以访问Lantern应用程序并提交log/ 报告。

go to Support (转到支持)
select Report an Issue (选择报告问题)
choose issue type (选择问题类型)
include additional details (包括额外的详细信息)
hit Send Report (点击发送报告)

当您在app提交问题报告后您会收到邮件. 邮件标题上有 8 位数号码. 请回来这里留言您的log/ticket号码. 因为我需要拿您的反馈报告号码来和科技团队沟通您遇到的问题. 不然我无法提交您遇到的问题给科技团队. 不好意思, 需要您的帮忙. 谢谢!

有了这些资料，我们科技团队便可以尽快解决问题。您可以在留言区给我知道。谢谢。

**Comment by Dionysus199404** [comment]
提交问题后邮件回复我的号码{73791390}

---- 回复的原邮件 ----
| 发件人 | Mike ***@***.***> |
| 发送日期 | 2024年06月21日 17:04 |
| 收件人 | getlantern/lantern-forum-cn ***@***.***> |
| 抄送人 | 杨雨涵 ***@***.***>,
Mention ***@***.***> |
| 主题 | Re: [getlantern/lantern-forum-cn] 更新lantern版本后无法登录 (Discussion #628) |

@***@***.***
您好，谢谢您的反馈。蓝灯已经在正视这个问题了。 我们正在修复问题和进行优化。请您耐心等待。非常抱歉影响了您的体验感。谢谢您的等待与支持。

您可以帮助我们在应用程序中提交报告您的问题吗？您可以访问Lantern应用程序并提交log/ 报告。

go to Support (转到支持)
select Report an Issue (选择报告问题)
choose issue type (选择问题类型)
include additional details (包括额外的详细信息)
hit Send Report (点击发送报告)

当您在app提交问题报告后您会收到邮件. 邮件标题上有 8 位数号码. 请回来这里留言您的log/ticket号码. 因为我需要拿您的反馈报告号码来和科技团队沟通您遇到的问题. 不然我无法提交您遇到的问题给科技团队. 不好意思, 需要您的帮忙. 谢谢!

有了这些资料，我们科技团队便可以尽快解决问题。您可以在留言区给我知道。谢谢。

**Comment by ChaoQunPeng** [comment]
无法发送报告







pengchaoqun
***@***.***



&nbsp;




------------------&nbsp;原始邮件&nbsp;------------------
发件人:                                                                                                                        "getlantern/lantern-forum-cn"                                                                                    ***@***.***&gt;;
发送时间:&nbsp;2024年6月21日(星期五) 下午5:04
***@***.***&gt;;
***@***.******@***.***&gt;;
主题:&nbsp;Re: [getlantern/lantern-forum-cn] 更新lantern版本后无法登录 (Discussion #628)





 
@Dionysus199404 @ChaoQunPeng
 您好，谢谢您的反馈。蓝灯已经在正视这个问题了。 我们正在修复问题和进行优化。请您耐心等待。非常抱歉影响了您的体验感。谢谢您的等待与支持。
 
您可以帮助我们在应用程序中提交报告您的问题吗？您可以访问Lantern应用程序并提交log/ 报告。
 
go to Support (转到支持)
 select Report an Issue (选择报告问题)
 choose issue type (选择问题类型)
 include additional details (包括额外的详细信息)
 hit Send Report (点击发送报告)
 
当您在app提交问题报告后您会收到邮件. 邮件标题上有 8 位数号码. 请回来这里留言您的log/ticket号码. 因为我需要拿您的反馈报告号码来和科技团队沟通您遇到的问题. 不然我无法提交您遇到的问题给科技团队. 不好意思, 需要您的帮忙. 谢谢!
 
有了这些资料，我们科技团队便可以尽快解决问题。您可以在留言区给我知道。谢谢。
